### PR TITLE
Feature/asb 2261 content audit accessibility cookies and contact us

### PR DESCRIPTION
**Pull request by cprieststephens**
## Describe your changes

2261: https://pins-ds.atlassian.net/jira/software/c/projects/ASB/boards/166?selectedIssue=ASB-2261
Content update for accessibility, cookies and contact pages.
Unit tests and e2e tests have been updated accordingly.

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by christopher-semmens — APPROVED** [review]

File: packages/forms-web-app/src/config.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ module.exports = {
 		supportTeamPhoneNumber: '0303 444 5000',
 		pressAndMediaPhoneNumber: '0303 444 5004',
 		pressAndMediaEmailAddress: 'press.office@planninginspectorate.gov.uk',
-		contactUsFormURL: 'https://contact-us.planninginspectorate.gov.uk/hc/en-gb/requests/new'
+		contactUsFormURL: 'https://contact-us.planninginspectorate.gov.uk/hc/en-gb/requests/new',
+		openingHours: '9:00 to 17:00'
```
Comment (emil-placheta) [inline packages/forms-web-app/src/config.js:80]: if possible, try to be a bit more descriptive with variable names ( within reason ofc ;) ), so other devs have some context on what information is held by it. `openingHours` is a bit vague, `csOpeningHours` is only a bit longer but it we can immediately infer it relates to custommer service opening times.

**Review by emil-placheta — CHANGES_REQUESTED** [review]
Just few notes

**Pull request merged**